wgpu/src/api/render_pipeline.rs

```diff
@@ -229,7 +229,7 @@ pub struct RenderPipelineDescriptor<'a> {
     pub fragment: Option<FragmentState<'a>>,
     /// If the pipeline will be used with a multiview render pass, this indicates how many array
     /// layers the attachments will have.
-    pub multiview: Option<NonZeroU32>,
+    pub multiview_mask: Option<NonZeroU32>,
```

Comment:
doc comment is a bit out of date. Still stalks about a count rather than a mask.